Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -773,9 +773,9 @@ void JitArm64::dcbx(UGeckoInstruction inst)
   // The first two function call arguments are already in the correct registers
   MOVP2R(ARM64Reg::X2, &m_system.GetJitInterface());
   if (make_loop)
-    MOVP2R(ARM64Reg::X8, &JitInterface::InvalidateICacheLinesFromJIT);
+    MOVP2R(ARM64Reg::X8, &JitInterface::InvalidateICacheLinesFromJitArm64);
   else
-    MOVP2R(ARM64Reg::X8, &JitInterface::InvalidateICacheLineFromJIT);
+    MOVP2R(ARM64Reg::X8, &JitInterface::InvalidateICacheLineFromJitArm64);
   BLR(ARM64Reg::X8);
```

Comment:
I don't know when this code dates from, but it seems very odd to use an indirect branch here when there is 0 runtime dependency for which function is being used (it's only codegen dependent, i.e. X8 ends up being a constant).

Reply:
Is there a different way to jump to an arbitrary 64-bit memory address on ARM?

Reply:
If the function is within 128 MB, you can use a simple `BL` instruction, but otherwise you have to use this approach. It's a limitation of how many address bits you can squeeze into one 32-bit instruction.